vllm/worker/vineyard_llm_cache.py

```diff
@@ -198,6 +198,9 @@ def prefetch_seq_kv_caches(
         duration = time.perf_counter() - start_time
         self.metrics.time_query.append(duration)
         self.metrics.normalized_time_query.append(duration/len(tokens))
+        # no need to minus 1 one more time. matched = min(matched, token_chunk_size - 1)
```

Comment:
This line of comment seems self-conflicting, because the code indeed is performing a minus one. It indeed is not "one more time", but that's probably due to the "minus one" above is commented. Could you please reconcile the comment and code here?

Reply:
updated